src/lib/variants.ts

```diff
@@ -346,6 +357,12 @@ export const findTestAndVariant = (tests: Test[], targeting: EpicTargeting): Res
         filters.every(filter => {
             const got = filter.test(test, targeting);
 
+            if (debug[test.name]) {
```

Comment:
Maybe you intended on wrapping this block in a `if (includeDebug) {` condition? As currently the `debug` object is always populated and returned in the JSON - the switch/boolean logic no applied as expected

Reply:
Or maybe you intended on doing that check in the route controller? Although it feels a bit wasteful to do that check so late.

Reply:
yeah this is a mistake. I've changed this. Now we always build up the debug object, but only return it if required. (I don't think the work to build it up is significant, but the resulting JSON will impact performance more.)